src/app/app.component.css

```diff
@@ -1870,3 +1870,7 @@ No Data Messaging
       margin: 0;
   }
 }
+
+:focus-visible {
```

Comment:
Not sure what part of the app promped this since this style is app-wide - was it solely in the directory/sidebar?. 1. Can we include this style only in the parts of the app where this is the most egregious and 2. can we just use the native browser focus color like what's seen on form fields?